packages/block-editor/src/store/reducer.js

```diff
@@ -1981,11 +1981,20 @@ export function blockEditingModes( state = new Map(), action ) {
 	switch ( action.type ) {
 		case 'SET_BLOCK_EDITING_MODE':
 			return new Map( state ).set( action.clientId, action.mode );
+		case 'SET_BLOCK_EDITING_MODES':
+			return new Map( [ ...state, ...action.modes ] );
```

Comment:
I think this is a leftover reducer logic.

Reply:
yes you are correct.